pyroclast/prototype/model.py

```diff
@@ -2,9 +2,10 @@
 import tensorflow as tf
 
 from pyroclast.prototype.prototype_layer import PrototypeLayer
+from pyroclast.common.feature_classifier import FeatureClassifierMixin
 
 
-class ProtoPNet(tf.Module):
+class ProtoPNet(tf.Module, FeatureClassifierMixin):
```

Comment:
This should read (FeatureClassifierMixin, tf.Module). Base classes should be rightmost when using mixins.